catalogs/usfimr-s1/build_mldata_catalog.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+
+import pystac
+
+import numpy as np
+from stac_utils.s3_io import register_s3_io
+from sklearn.model_selection import train_test_split
```

Comment:
I don't think scikit learn is in the requirements.txt for the parent docker container in ./catalogs. If it isn't don't forget to add it along with any other reps you need for this PR